src/main/java/co/norse/chat/controller/UserController.java

```diff
@@ -0,0 +1,61 @@
+package co.norse.chat.controller;
+
+import co.norse.chat.User;
+import co.norse.chat.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class UserController {
+    @Autowired
+    UserService userService;
+
+    @GetMapping(value = "/users")
+    public ResponseEntity<List<User>> getAllUsers() {
```

Comment:
actually you don't need ResponseEntity, you can return just list. Plus if no users in system should be returned 200ok and empty list, thus your method will look just
` @GetMapping(value = "/users")
    public List<User> getAllUsers() {
        return userService.findAllUsers();
    }`